predefine.cmake

```diff
@@ -0,0 +1,82 @@
+set(CC_PLATFORM_OPENHARMONY 14)
+add_definitions(-DCC_PLATFORM_OPENHARMONY=${CC_PLATFORM_OPENHARMONY})
+
+set(CC_PLATFORM ${CC_PLATFORM_OPENHARMONY})
+add_definitions(-DOPENHARMONY=${CC_PLATFORM_OPENHARMONY})
+add_definitions(-DCC_PLATFORM=${CC_PLATFORM})
+set(CMAKE_CXX_FLAGS "-fvisibility=hidden -fvisibility-inlines-hidden ${CMAKE_CXX_FLAGS}")
+if("${OHOS_ARCH}" STREQUAL "armeabi-v7a")
+    set(CMAKE_CXX_FLAGS "-march=armv7a ${CMAKE_CXX_FLAGS}")
+endif()
+
+MESSAGE(STATUS "platform: ${CMAKE_SYSTEM_NAME}")
+
+################################# cc_set_if_undefined ###################################
```

Comment:
I think it is not needed as v2.x doesn't use cmake.

Reply:
fixed